### DISCUSSION TITLE
Multiple webpack encore config

### DISCUSSION THREAD

**Discussion by Dolu89** [adonisjs/core#3734] [Help]
Hello,

I have an issue trying to have 2 webpack encore config (for internal JS and exposed JS). I have seen Webpack encore supports it https://symfony.com/doc/current/frontend/encore/advanced-config.html#defining-multiple-webpack-configurations
But in adonis, when I'm launching the dev server I got this issue `[ encore ] [webpack-cli] Error: Unique ports must be specified for each devServer option in your webpack configuration. Alternatively, run only 1 devServer config using the --config-name flag to specify your desired config.`
My webpack.config.js is very basic

```js
// ... configuration provided by adonis
const config = Encore.getWebpackConfig()
config.infrastructureLogging = {
  level: 'warn',
}
config.stats = 'errors-warnings'
config.name = 'privateConfig';

Encore.reset()
//... new Encore config
const publicConfig = Encore.getWebpackConfig()
publicConfig.infrastructureLogging = {
  level: 'warn',
}
publicConfig.stats = 'errors-warnings'
publicConfig.name = 'publicConfig';

module.exports = [config, publicConfig]
```
Any chance I just missed something? Or are multi config not supported in adonis?

Thanks

**Comment by filipebraida** [comment]
I'm having the same issue. I wish one of the assets wasn't being generated through versioning.
I tried a few possibilities and couldn't get any of them to work. The problem is that you can't define the port for webpack.
https://github.com/symfony/webpack-encore/issues/923
